metis-enrichment/metis-enrichment-service/src/main/java/eu/europeana/enrichment/service/RedisInternalEnricher.java

```diff
@@ -201,13 +193,14 @@ private void loadEntities(EntityType entityType) {
     for (MongoTerm term : terms) {
       loadEntity(entityType, term, jedis);
       i++;
-      if (i % 100 == 0) {
+      final int countOfItemsCollectedToLog = 100;
```

Comment:
Shouldn't this be a constant outside this method?

Reply:
Because it's just about logging I find this not, so I find both options of the same value. What do you think?

Reply:
I don't know... I'm just used to putting it outside. Also for when I have to create another piece of code that does logging in that class I will easily see that this setting is already there. But maybe I'm thinking this way just because I'm used to it.

Anyway, just do what you think is right. And feel free to merge the code when you're done.